### PR TITLE
Added a bond variable and related tests for the userInteractionEnable…

### PR DESCRIPTION
**Pull request by murrayds**
Added a bond variable and related tests for the userInteractionEnabled variable of UIView. Format follows existing code and testing runs successfully.

### CODE REVIEW

File: Bond/iOS/Bond+UIView.swift [hunk 1]
```diff
@@ -71,4 +72,18 @@ extension UIView {
       return d
     }
   }
+    
+  public var dynUserInteractionEnabled: Dynamic<Bool> {
+    if let d: AnyObject = objc_getAssociatedObject(self, &userInteractionEnabledUIView) {
+      return (d as? Dynamic<Bool>)!
+    } else {
+      let d = InternalDynamic<Bool>(self.userInteractionEnabled)
+      let bond = Bond<Bool>() { [weak self] v in if let s = self { s.userInteractionEnabled = v } }
```
Comment (ivanmoskalev) [inline Bond/iOS/Bond+UIView.swift]: Why not just `self?.userInteractionEnabled = v`?

**Comment by ivanmoskalev** [comment]
I think this can be merged.
@srdanrasic, can I merge the PRs, or you'd prefer to do it yourself?

File: Bond/iOS/Bond+UIView.swift (hunk 1 above)
Comment (murrayds) [inline Bond/iOS/Bond+UIView.swift]: @ivanmoskalev Sorry for the delay in responding to this comment. The updated statement you recommend works fine however it is inconsistent with the implementation of other dynamic variables in the project. I do believe that all dynamic variables should be updated however that task is outside the scope of this branch.

Comment (ivanmoskalev) [inline Bond/iOS/Bond+UIView.swift]: I agree on that :)

Comment (murrayds) [inline Bond/iOS/Bond+UIView.swift]: @ivanmoskalev Thanks for the input, I created an issue for the proposed changes so that discussion can take place. I might take ownership of the issue and make the change in the near future.

**Comment by srdanrasic** [comment]
Thank you @murrayds for the PR and @ivanmoskalev for the review. I'll merge it.
